prime-router/src/main/kotlin/LookupTable.kt

```diff
@@ -97,6 +97,17 @@ class LookupTable(
         }
     }
 
+    /**
+     * Get the set of distinct values in a column in the table.
+     */
+    fun getDistinctValuesInColumn(selectColumn: String): Set<String> {
+        val selectColumnNumber = headerIndex[selectColumn.toLowerCase()] ?: return emptySet()
+        return table
+            .map { row -> row[selectColumnNumber] }
+            .filter { !it.equals(selectColumn) }   // terrible hack to remove the column header.
+            .toSet()
```

Comment:
perhaps a .drop(1) would be a clearer way to remove the header row.

Reply:
Done!